modules/storage-persistent-storage-reclaim-manual.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+
+[id="reclaim-manual_{context}"]
+= Manually reclaiming a PV
+
+When a PersistentVolumeClaim (PVC) is deleted, the PersistentVolume (PV) still exists and is considered "released". But the PV is not yet available for another claim because the previous claimant's data remains on the volume.
+
+To manually reclaim the PV as a cluster administrator:
+
+. Delete the PV.
++
+[source,terminal]
+----
+$ oc delete <pv-name>
+----
++
+The associated storage asset in the external infrastructure, such as an AWS EBS, GCE PD, Azure Disk, or Cinder volume, still exists after the PV is deleted.
+
+. Manually clean up the data on the associated storage asset.
+
+. Manually delete the associated storage asset, or if you want to reuse the same storage asset, create a new PV with the storage asset definition.
+
+The retained PV is now available for use by another PVC.
```

Comment:
I suggest keeping the same "reclaimed" wording because its meaning is slightly different than "retained."